content/graphing/dashboards/widgets.md

```diff
@@ -228,7 +230,11 @@ Example of Image widget for the [API](/api/#screenboards)
 ## Note
 *Supported on Screenboards only*
 
-Note is similar to Free Text, but allows for more formatting options. Also, an arrow can be added to the text box that appears on the dashboard. This is commonly used to document the structure of the dashboard.
+Note is similar to Free Text, but allows for more formatting options: 
+
+* An arrow can be added to the text box that appears on the dashboard. This is commonly used to document the structure of the dashboard.  
+
+* [Use href to create internal links in Datadog](http://cl.ly/310B2k322j0W).  
```

Comment:
I really like the screen recording! 🙂 

I just had one thought: do you think we could mention it's a screen recording in the link? Right now it's not apparent what the link goes to (a blog post, a faq, etc.).

Would we be able to do something like this:
```markdown
* [Screen recording showing how to use `href` to create internal links in Datadog](http://cl.ly/310B2k322j0W).
```

Reply:
👍